ElectronicObserverTypes/SliderUpgradeLevel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace ElectronicObserverTypes;
+public enum SliderUpgradeLevel
+{
+	/// <summary>Never</summary>
+	[Display(ResourceType = typeof(Properties.UpgradeLevel), Name = "Never")]
+	Never = 0,
```

Comment:
Maybe it might be smarter to have `Never` as the highest value?
Also, slidering from 5 is also a thing.

Reply:
Changed the enum logic and display